### PR TITLE
Inventory service with dedicated db

### PR DESCRIPTION
**Pull request by Jozefiel**

### CODE REVIEW

File: config/dev_settings.txt [hunk 1]
```diff
@@ -77,4 +77,11 @@ IV_ULIMIT_NOFILE_HARD=65535
...
-IV_RES_LIMIT_MEM=128M
\ No newline at end of file
+IV_RES_LIMIT_MEM=128M
+
+IP_ULIMIT_NOFILE_SOFT=65535
+IP_ULIMIT_NOFILE_HARD=65535
+IP_ULIMIT_NPROC_SOFT=5000
+IP_ULIMIT_NPROC_HARD=5000
+IP_RES_LIMIT_CPUS=0.5
+IP_RES_LIMIT_MEM=128M
```
Comment (marosmars) [inline config/dev_settings.txt:87]: does this work ? with 128GB RAM ? Seems like too little for psql to work with

File: config/krakend/partials/tenant_headers.tmpl [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (marosmars) [inline config/krakend/partials/tenant_headers.tmpl:1]: I would rename the file to default_tenant_headers

File: config/dev_settings.txt (hunk 1 above)
Comment (Jozefiel) [reply config/dev_settings.txt]: Memory and CPU limit was increased

File: config/krakend/partials/tenant_headers.tmpl (hunk 2 above)
Comment (Jozefiel) [reply config/krakend/partials/tenant_headers.tmpl]: name changed

**Review by marosmars — APPROVED** [review]

**Pull request merged**